.vib/schema-registry/goss/goss.yaml

```diff
@@ -0,0 +1,8 @@
+file:
+  /opt/bitnami/schema-registry/etc/schema-registry/schema-registry.properties:
+    mode: "0664"
+    filetype: file
+    exists: true
+    contains:
+      - "listeners = {{ .Vars.listeners }}"
+      - "avro.compatibility.level = {{ .Vars.avroCompatibilityLevel }}"
```

Comment:
I have noticed that this chart does also create a serviceAccount and automounts its secret by default. Hence, I think it would be worth adding a test to check this is the case. See:

https://github.com/bitnami/charts/blob/2a88f4661df35140dc39c72e7677357de764bcc3/.vib/kafka/goss/goss.yaml#L20-L23

as an example